node/test/chan_drain.js

```diff
@@ -739,11 +734,24 @@ function checkFinish(assert, err, cluster, finishCount) {
                 {peers: [{connections: []}]}
             ]
         });
-    }
 
-    if (finishCount <= 0) {
         assert.end();
     }
 
     return finishCount;
 }
+
+function waitForConnRemoved(count, cluster, callback) {
```

Comment:
I actually tried it first, the last two test cases timeout, because only 2 out of 6 connections are expected to be removed.